app/views/layouts/_javascripts.html.erb

```diff
@@ -71,41 +71,13 @@
           }
       }
 
-      // For browsers that don't support localStorage according to our check,
-      // make its functions handle cookies instead.
-      // Credit: https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Local_storage
-      if (!isSupported()) {
-        // 20 years is the Rails maximum and what we previous used.  
-        var expiryDate = new Date;
-        expiryDate.setFullYear(expiryDate.getFullYear() + 20);
-
-        // window.localStorage is read only, ergo we must delete it for this to
-        // work in instances like Safari 9 private mode.
-        delete window.localStorage;
-        window.localStorage = {
-          getItem: function (sKey) {
-            if (!sKey || !this.hasOwnProperty(sKey)) { return null; }
-            return unescape(document.cookie.replace(new RegExp("(?:^|.*;\\s*)" + escape(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=\\s*((?:[^;](?!;))*[^;]?).*"), "$1"));
-          },
-          key: function (nKeyId) {
-            return unescape(document.cookie.replace(/\s*\=(?:.(?!;))*$/, "").split(/\s*\=(?:[^;](?!;))*[^;]?;\s*/)[nKeyId]);
-          },
-          setItem: function (sKey, sValue) {
-            if(!sKey) { return; }
-            document.cookie = escape(sKey) + "=" + escape(sValue) + "; expires=" + expiryDate.toUTCString() + "; path=/";
-            this.length = document.cookie.match(/\=/g).length;
-          },
-          length: 0,
-          removeItem: function (sKey) {
-            if (!sKey || !this.hasOwnProperty(sKey)) { return; }
-            document.cookie = escape(sKey) + "=; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/";
-            this.length--;
-          },
-          hasOwnProperty: function (sKey) {
-            return (new RegExp("(?:^|;\\s*)" + escape(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=")).test(document.cookie);
-          }
-        };
-        window.localStorage.length = (document.cookie.match(/\=/g) || window.localStorage).length;
+      function acceptTOS() {
+        if (isSupported()) {
+          localStorage.setItem("accepted_tos", "<%= @current_tos_version %>");
+        } else {
+          // 20 years is the Rails maximum and what we previously used. 
```

Comment:
"and what we previously used" is not necessary, we can just say it's the same as Rails' permanent cookies.